dist/templates/k8s.ovn.org_egressfirewalls.yaml.j2

```diff
@@ -79,6 +79,8 @@ spec:
                     to:
                       description: to is the target that traffic is allowed/denied
                         to
+                      maxProperties: 1
+                      minProperties: 1
```

Comment:
From what I can tell it is applying the min and max properties to the type (allow | deny) property as well as the property that effects the location (dnsName, nodeSelector, cidrSelector)

Reply:
@JacobTanenbaum but that's surprising...so this is what is happening on https://issues.redhat.com/browse/OCPBUGS-8280 as well which means our CNO PR also has this issue :/ ?

However I am not able to see how this is happening; guess I am blind. If I align the lines
to:
  minProperties:
  maxProperties:
type:
are under egress, I see the minProperties and maxProperties only being applied to `to`.. right? what am I missing..

Reply:
its because on ovnk updating the object in kapi, nodeSelector was not set to nil, it was set to a selector struct that had nil match expr and nil label. Changed it to a pointer and now it works.